packages/fields/src/fields/template/template-edit.tsx

```diff
@@ -0,0 +1,210 @@
+/**
```

Comment:
For file names: we could simplify to `edit` or `view` if they are within a field folder.